templates/expo-template-tabs-react-navigation/.gitignore

```diff
@@ -36,3 +36,7 @@ yarn-error.*
 *.tsbuildinfo
 
 app-example
+
+# generated native folders
```

Comment:
noticing that this template file is called **.gitignore** - it should be **gitignore** to work with `create-expo` properly

Reply:
I had the same observation! Just merged that PR.